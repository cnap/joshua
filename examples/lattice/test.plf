((('ein',0.1,1),('dieses',0.2,1),('haus',0.4,2),),(('haus',0.8,1),),)
