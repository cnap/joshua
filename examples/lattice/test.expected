0 ||| this house ||| -1.866 -0.000 -0.000 -0.021 -0.869 -1.000 ||| -2.008
0 ||| this house ||| -1.866 -0.000 -0.000 -0.021 -0.869 -1.000 ||| -2.008
0 ||| this house ||| -1.866 -0.022 -0.000 -0.021 -0.869 -1.000 ||| -2.031
0 ||| this house ||| -1.866 -0.022 -0.000 -0.021 -0.869 -1.000 ||| -2.031
0 ||| a house ||| -1.888 -0.000 -0.228 -0.021 -0.869 -0.900 ||| -2.044
0 ||| a house ||| -1.888 -0.022 -0.228 -0.021 -0.869 -0.900 ||| -2.067
0 ||| a house ||| -1.888 -0.022 -0.228 -0.021 -0.869 -0.900 ||| -2.067
0 ||| a house ||| -1.888 -0.079 -0.228 -0.021 -0.869 -0.900 ||| -2.123
0 ||| house ||| -2.555 -0.022 -0.000 -0.021 -0.434 -0.400 ||| -2.553
0 ||| a small house ||| -2.279 -0.778 -0.228 -1.343 -1.303 -0.900 ||| -3.440
0 ||| a small house ||| -2.279 -1.301 -0.228 -1.343 -1.303 -0.900 ||| -3.963
0 ||| a small house ||| -2.279 -1.301 -0.228 -1.343 -1.303 -0.900 ||| -3.963
0 ||| this small house ||| -3.166 -1.301 -0.000 -1.343 -1.303 -1.000 ||| -4.836
0 ||| this small house ||| -3.166 -1.301 -0.000 -1.343 -1.303 -1.000 ||| -4.836
0 ||| small house ||| -3.877 -1.301 -0.000 -1.343 -0.869 -0.400 ||| -5.381
